ee/codegen/src/context/node-context/create-node-context.ts

```diff
@@ -48,9 +50,19 @@ export async function createNodeContext(
           });
         }
         case "DEPLOYMENT": {
+          const { releaseTag, workflowDeploymentId } = subworkflowNodeData.data;
+          const workflowDeploymentHistoryItem =
+            await new WorkflowDeploymentsClient(
+              {}
+            ).workflowDeploymentHistoryItemRetrieve(
+              releaseTag,
+              workflowDeploymentId
```

Comment:
woah, kinda wild that fern generates it in this order

Reply:
Yeah 😕 the worst part is if they “fixed” it to be in the reverse order, it’d be a breaking logical change uncaught by type checks 😵‍💫

Reply:
Ah I suspect it's because our OpenAPI spec has it in this order: https://github.com/vellum-ai/vellum-client-generator/blob/18e4eb34c2e87b86a86f4af45ea7890485f833f5/fern/openapi/openapi.yml#L269

Given that these are new endpoints, I'm gonna try to make this breaking change asap before people start relying on them.

Reply:
> it looks like drf spectacular generates the order of query parameters in the spec alphabetically

can we override this? I think fern following the order of the spec is "correct"

Reply:
Not that I can find.